### ISSUE TITLE
reconstruction_system error

### ISSUE THREAD

**Issue by azubu23202869** [intel-isl/Open3D#2575]
- Operating system:  Windows 10 64-bit
- Python version: Python 3.6
- Open3D version: 0.11.1
- Is this remote workstation?: no
- How did you install Open3D?: conda

```diff
numpy = 1.19.1
joblib=0.17.0
```
CPU:R5 3600    
RAM:16GB

if realsense_recoder get >700 image 

```diff
cd reconstruction_system
python run_system.py config/realsense.json --make 
```

the following problems will occur when running halfway

```diff
This could be caused by a segmentation fault while calling the function or by an excessive memory usage causing the Operating System to kill the worker.
```

realsense.json and camera_intrinsic.json are both default settings

I have tried creating new conda env and updating open3d

**Comment by ashura1234** [comment]
Same here, can't even run the tutorial.

**Comment by theNded** [comment]
This has been reported and intensively discussed in #2107. It has been fixed in #2567. Please try the up-to-date master branch for now, and we will be releasing a patch soon.
